src/risk-risiko.py

```diff
@@ -0,0 +1,69 @@
+"""
+  Write a program that simulates a risk/risiko fight using 6 dices.
+  How does it work?
+  When a player attacks another player he uses 3 dices, 
+  the red is always the attacker and the blue is the defender.
+  You have to compare the dice with the highest number to simulate 
+  the fight.
+  N = first highest number
+  M = second highest number
+  O = third highest number
+  If the numbers are equal, the defensor (blue) wins.
+  Output:
+  Red dices:
+  6 (N)
+  3 (M)
+  2 (O)
+  Blue dices:
+  5 (N)
+  3 (M)
+  1 (O)
+    R    B
+  N 6 vs 5 => red win
+  M 3 vs 3 => blue win
+  O 2 vs 1 => red win
+"""
+import random
+
+def main():
+    
+    print ("Rolling attacker (red) dices.. \n")
+    atk_dice_n = random.randint(1,6)
```

Comment:
crea una funzione che effettua il lancio del dado e ritorna il valore invece di ripetere la chiamata a randint con gli stessi parametri

Reply:
vedendo il codice sotto, credo che sia comodo avere una funzione che calcoli il valore di n dadi:

random_dice_run(n)

e ritorna una tupla di n elementi.

potresti in questo modo riutilizzare la stessa funzione anche per il difensore, tanto ciò che cambia non è il lancio di dadi ma il confronto.